### PR TITLE
Reenable frux

### PR DESCRIPTION
**Pull request by dabbler0**

### CODE REVIEW

File: src/ConnectionMonitor.ts [hunk 1]
```diff
@@ -1,18 +1,9 @@
...
-const Qualities = [
-  "excellent",
-  "good",
-  "average",
-  "poor",
-  "bad",
-  "unknown",
-] as const;
-export type Quality = typeof Qualities[number];
+import { ConnectionStateQuality } from "./API";
```
Comment (jackofsometrade) [inline src/ConnectionMonitor.ts:3]: ❤️

**Review by jackofsometrade — APPROVED** [review]
LGTM!

File: src/RoomClient.ts [hunk 2]
```diff
@@ -210,6 +212,18 @@ class RoomClient {
     if (selfReport) {
       // Update local latest status
       this.user.status = selfReport.status;
+
+      // Respond to frux
+      if (this.fruxEnabled) {
```
Comment (cainlevy) [inline src/RoomClient.ts:217]: is it sufficient to simply control the publishing of `connectionState` events? it seems like this code might be safe to execute even when frux is disabled.

File: src/RoomClient.ts [hunk 3]
```diff
@@ -428,6 +441,14 @@ class RoomClient {
     )
       return;
 
+    // Do not allow resuming video when connection is bad
+    if (
+      label === ProducerLabel.video &&
+      this.fruxEnabled &&
+      this.user.connectionState.videoDisabled
```
Comment (cainlevy) [inline src/RoomClient.ts:448]: is it possible for "videoDisabled" to be true for another reason than a poor connection quality? that is, is it redundant to also check `this.fruxEnabled`?

**Review by cainlevy — APPROVED** [review]
looks ready to me.

my attached code comments come from reflecting on @jackofsometrade's point about naming. it could be nice to describe the behavior rather than an opaque acronym, but it's hard to know what to suggest without understanding the behavior correctly.

for example, would it be accurate to call this `enableQualityMonitoring` or `publishConnectionState`? or should we consider removing this option from the client and controlling the feature entirely from the server?

**Pull request merged**